samples/extensions/SampleExtensions.Tests/SampleExtensions.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="microsoft.dotnet.interactive.csharp" Version="1.0.0-beta.23562.1" />
     <PackageReference Include="microsoft.dotnet.interactive.fsharp" Version="1.0.0-beta.23562.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.10.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
Bumping this version should be done using the version in the prop file in root directory

Reply:
Not for the sample project.